packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamWizard.tsx

```diff
@@ -121,51 +121,10 @@ const ChallengeTeamWizard: React.FunctionComponent<
   const { data: userProfile } = useGetCurrentUserProfile()
   // Retrieve the challenge associated with the projectId passed through props
   const { data: challenge } = useGetEntityChallenge(projectId)
-  const participantTeamId: string = challenge
-    ? challenge.participantTeamId
-    : EMPTY_ID
-  const userId = userProfile ? userProfile.ownerId : EMPTY_ID
-
-  // Verify that user is a member of the participant team
-  const { data: challengeTeamMembershipStatus } = useGetMembershipStatus(
-    participantTeamId,
-    userId,
-  )
-  useEffect(() => {
-    if (
-      challengeTeamMembershipStatus &&
-      !challengeTeamMembershipStatus?.isMember &&
-      accessToken
-    ) {
-      addTeamMemberAsAuthenticatedUserOrAdmin(
-        participantTeamId,
-        userId,
-        accessToken,
-      )
-        .then(() => {
-          queryClient.invalidateQueries(
-            keyFactory.getMembershipStatusQueryKey(participantTeamId, userId),
-          )
-          queryClient.invalidateQueries(
-            keyFactory.getIsUserMemberOfTeamQueryKey(participantTeamId, userId),
-          )
-        })
-        .catch(error => {
-          setErrorMessage(error.reason)
-        })
-    }
-  }, [
-    accessToken,
-    participantTeamId,
-    userId,
-    challengeTeamMembershipStatus,
-    queryClient,
-    keyFactory,
-  ])
```

Comment:
This effect 'magically' added the user to the participant team when this wizard was shown. Having this operation in an effect, triggering it simply by rendering this component, is remarkably unintuitive. Since registration now happens when the user clicks 'Register' in the new component, this effect can be removed.